content/ko/docs/contribute/style/content-organization.md

```diff
@@ -0,0 +1,147 @@
+---
+title: 콘텐츠 구성
+content_type: concept
+weight: 90
+---
+
+<!-- overview -->
+
+이 사이트는 Hugo를 사용한다. Hugo에서, [콘텐츠 구성](https://gohugo.io/content-management/organization/)은 핵심 개념이다.
+
+<!-- body -->
+
+{{% note %}}
+**Hugo 팁:** 콘텐츠 편집 세션을 시작할 때는 `hugo server --navigateToChanged`로 Hugo를 실행한다.
+{{% /note %}}
+
+## 페이지 목록
+
+### 페이지 순서
+
+문서 사이드 메뉴, 문서 페이지 브라우저 등은 Hugo의 기본 정렬 순서를 
+사용하여 나열되며, 이 정렬 순서는 weight(1부터 시작), date(최신 항목 우선), 
+그리고 link title 순서이다.
+
+이를 바탕으로, 페이지나 섹션을 위로 올리고 싶다면, 페이지의 전문에 weight 값을 설정한다.
```

Comment:
@kamothi Please discuss with @developowl how to best translate "front matter" (see also https://github.com/kubernetes/website/pull/53762 )